spec/shared/room_spec.rb

```diff
@@ -0,0 +1,383 @@
+shared_examples 'room_behavior' do |room_type, query:|
```

Comment:
Yeah, I would say for the sake of Ruby 2.0 support I'd add the default.

Also, I would also suggest this file be renamed to remove the `_spec`. Maybe match the shared example block name? `room_behaviours.rb` ? Otherwise the default rspec file matcher will try and 'run' the shared examples on its own (not that this will have any effect, but no point it trying in the first place).